lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -663,6 +664,9 @@ func convertContainerToContainerInfo(c *exec.Container) *models.ContainerInfo {
 			ep.Ports = append(ep.Ports, endpoint.Ports...)
 		}
 
+		macaddress, _ := container.GetContainerVM().GetMacAddressBasedSpecifiedNetworkName(context.Background(), endpoint.Assigned.IP.String())
```

Comment:
I think we should have callers of `convertContainerToContainerInfo` pass in a context instead of using `context.Background()`. It looks like both callers have an operation (a context that includes debugging information); if we use that instead of `context.Background()`, it will be easier to associate any log messages from `GetMacAddressBasedSpecifiedNetworkName` with the operation that caused it to be called.

Reply:
done. I pass the trace.operation